src/controllers/wallet/p2sh/tx-proposal.controller.js

```diff
@@ -51,7 +51,48 @@ async function buildTxProposal(req, res) {
 
     res.send({ success: true, txHex: tx.toHex() });
   } catch (err) {
-    console.error(err);
+    res.send({ success: false, error: err.message });
```

Comment:
Should we keep the `console.error` message?
It's shows on the logs the reason for the error.

Reply:
Well, we can keep.